tutorials/preparing-the-vm.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Preparing the Virtual Machine
+---
+## Purpose
+This lesson describes the steps for configuring a virtual machine to run Kite example code on a Cloudera Quickstart VM.
+
+### Result
```

Comment:
The prerequisite for this lab is that the user has either VMWare or VirtualBox installed. Links to download those tools would be helpful.